docs/en/reference/sql/dql/NO_TABLE_SELECT_CLAUSE.md

```diff
@@ -14,25 +14,30 @@ SelectExpr    ::= ( Identifier '.' ( Identifier '.' )? )? '*'
      
 ```
 
-## SQL Statement Template
+## SQL Template
 
 ```sql
 SELECT const_expr [, const_expr ...];
 ```
 
 ## 2. SELECT Statement Elements
 
-| SELECT statement elements | state                | direction                                                         |
-| :------------- | ------------------- | :----------------------------------------------------------- |
-| Unlabeled SELECT statement | OnlineServing not supported | The no table Select statement calculates the constant expression operation list, and the expression calculation does not need to depend on the table and column |
+
+| `SELECT` Statement Elements | Offline Mode | Online Preview Mode | Online Request Mode | Illustrate                                                                                                                             |
```

Comment:
Either direction or illustrate is weird, you may simply change it to "Note"

Reply:
ok~I will change the name in all related docs